.github/workflows/release.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
           doxygen \
           libbrotli-dev \
           libgdk-pixbuf2.0-dev \
+          libgflag-dev \
```

Comment:
Should it be "libgflags-dev"?

Reply:
yes.
Thanks for fixing it in https://github.com/libjxl/libjxl/commit/a46cbcc65ca1f22820c25343330a441cf9f658e3 !